src/main/java/seedu/address/model/session/Subject.java

```diff
@@ -24,14 +24,33 @@ public Subject(String value) {
         this.value = value;
     }
 
+    public String getValue() {
+        return value;
```

Comment:
I believe `this.value` would be better? 🐷

Reply:
Amended in next commit!